python-avd/pyproject.toml

```diff
@@ -0,0 +1,41 @@
+[build-system]
+requires = ["setuptools>=61.2", "wheel"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name="pyavd"
+version = "4.0.0-dev9"
```

Comment:
please investigate ways of dynamically getting the version from the version.py created by `make dep`

Reply:
<AVD version><pyavd suffix>
like:
4.0.0-dev9-pyavd-alpha-1

Reply:
Fixed!